### PR TITLE
Added Compliance.color() + Compliance.badge_image_url() method

### LINKED ISSUE #301: Compliance add public method to get color of the badge
Relevant code:

https://github.com/fair-software/howfairis/blob/738f2f9a0c5fc4409199af9bb189bc86b8dfaa47/howfairis/compliance.py#L78-L87

### LINKED ISSUE #304: Add badge image url property to Compliance
Using howfairis as a library in fairtally I noticed the badge image url was needed and it had to be constructed in an awkward way at
https://github.com/fair-software/fairtally/blob/67e34548e535369d213cab19f2b4b425e9fb7fdb/fairtally/cli.py#L56-L57

It would be nice to have besides a Markdown or Restructured snippet for the badge to also have the raw image url and target url be part of the Compliance class interface.
When implemented fairtally can just request what it needs without resorting to duplicating code.

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #301
Fixes #304

**Instructions to review the pull request**

```ipython
In [1]: from howfairis import Compliance
In [2]: compliance = Compliance(True, True, True, True, True)
In [3]: compliance.color()
Out[3]: 'green'
In [4]: compliance.badge_image_url()
Out[4]: 'https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F-green'
```
With all recommendations compliant it should return green color and image URL should look like Image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Compliance.color() [#301](https://github.com/fair-software/howfairis/issues/301)
+- Compliance.badge_image_url() [#304](https://github.com/fair-software/howfairis/issues/304)
```
Comment (jspaaks) [inline CHANGELOG.md:11]: :+1:

**Review by jspaaks — CHANGES_REQUESTED** [review]
Small note about naming: I would have expected `.get_color()` or `.color` and same for the other method. Otherwise, a nice addition!

**Comment by sverhoeven** [comment]
> Small note about naming: I would have expected `.get_color()` or `.color` and same for the other method. Otherwise, a nice addition!

The surrounding methods dont start with `get_` so I stayed with the look and feel inside the Compliance class.

**Review by jspaaks — APPROVED** [review]
:baby_chick: :baby_chick: :baby_chick: :baby_chick: :baby_chick:

**Comment by sverhoeven** [comment]
Thanks for reviewing

**Pull request merged**